### PR TITLE
Milestonefour

### PR DESCRIPTION
**Pull request by JaredBronstein**
Inventory system and some Farm touch-ups

### CODE REVIEW

File: Assets/Scripts/Door.cs [hunk 1]
```diff
@@ -20,16 +61,40 @@ protected override void Awake()
...
         base.Awake();
         animator = GetComponent<Animator>();
+        InitialIsLocked();
+    }
+
+    private void InitialIsLocked()
+    {
+        if (key != null)
+            isLocked = true;
+
```
Comment (Aporoid) [inline Assets/Scripts/Door.cs]: Delete blank line

File: Assets/Scripts/Door.cs [hunk 2]
```diff
@@ -1,12 +1,53 @@
...
+    [Tooltip("Play this audio clip when the player interacts with a locked door without owning the key.")]
+    [SerializeField]
+    private AudioClip lockedAudioClip;
+
+    [Tooltip("Play this audio clip when the player opens the door.")]
+    [SerializeField]
+    private AudioClip openAudioClip;
+
+    // public override string DisplayText => isLocked ? lockedDisplayText : base.DisplayText;
```
Comment (Aporoid) [inline Assets/Scripts/Door.cs]: Consider removing this if the action is commented out and unnecessary. Also get rid of the accompanying blank line.

File: Assets/Scripts/InteractiveObject.cs [hunk 3]
```diff
@@ -6,11 +6,12 @@
```
Comment (Aporoid) [inline Assets/Scripts/InteractiveObject.cs:6]: Consider adding a summary for sake of clarity.

**Review by Aporoid — COMMENTED** [review]
Add some summaries and clean up empty lines, recommit and you're golden.

**Comment by Aporoid** [comment]
All good, free to merge.

**Review by Aporoid — APPROVED** [review]
Now it's good.

File: Assets/Scripts/Door.cs (hunk 2 above)
Comment (dantogno) [reply Assets/Scripts/Door.cs]: Yes, always remove commented out code!

**Pull request merged**